resourceDefinition.json

```
@@ -398,11 +398,11 @@
   {
     "name": "cosmosdb_account",
     "length": {
-      "min": 1,
-      "max": 63
+      "min": 3,
+      "max": 44
     },
-    "regex": "^(?=.{1,63}$)[a-z0-9][a-z0-9-_.]+[a-z0-9]$",
-    "scope": "resourceGroup",
+    "regex": "^(?=.{3,44}$)[a-z0-9][a-z0-9-]{2,}$",
```

```suggestion
    "regex": "^(?=.{3,44}$)[a-z0-9][a-z0-9-]{1,}[a-z0-9]$",
```

*See my previous comment*